<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editing_table</class>
 <widget class="QDialog" name="editing_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>22</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 22pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>EDIT TABLES</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>1001</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Table No.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_e1">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Seats</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_e2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Table Location</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_e3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Table type</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_e5">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QTextEdit" name="textEdit_1">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_change">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>610</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>610</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
